example/android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,5 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.6.1-all.zip
\ No newline at end of file
+# distributionUrl=https\://services.gradle.org/distributions/gradle-7.6.2-all.zip
+distributionUrl=file\:///home/sumit/softwares/gradle-7.6.2-all.zip
```

Comment:
is there a reason this change was committed with the PR? 
this obviously wont work on other devices.